nixos/modules/services/web-apps/openwebrx.nix

```diff
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.openwebrx;
+in
+{
+  options.services.openwebrx = with lib; {
+    enable = mkEnableOption "Enable OpenWebRX Web interface for Software-Defined Radios";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.openwebrx;
+    };
```

Comment:
When would this ever be used? Are there any other options other than `pkgs.openwebrx`?